src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -95,12 +95,12 @@ private void chekIndex(int index) {
     }
 
     private void setHead(T element) {
-        if (head == null) {
+        if (head != null) {
+            head = new Node<>(null, element, head);
```

Comment:
Розберись пліз що робить цей код))
```suggestion
            Node<T> newNode = new Node<>(null, element, head);
            head.prev = newNode;
            head = newNode;
```